### PR TITLE
Updated RPM to use systemd to remove daemonize dependency

### PR DESCRIPTION
**Pull request by jamtur01**
- Added pkg/rpm/riemann.service file
- Supports start, stop, reload

This deprecates the previous init.sh script that required daemonize.

This assumes your Red Hat-family system supports systemd.

### CODE REVIEW

**Comment by jamtur01** [comment]
Ping @mfournier for comment.

**Comment by faxm0dem** [comment]
as many folks still use EL6, the leiningen part would probably need a switch or autodetect

**Comment by aphyr** [comment]
If we need to support both EL6 and EL7 then I suggest we do that switching *inside* the package instead of as a lein task--I'd rather not add another build target. Could we use a postinstall script to detect systemd and copy the right script?

File: pkg/rpm/riemann.service [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Riemann
+After=network.target
+
+[Service]
+User=riemann
+EnvironmentFile=/etc/sysconfig/riemann
+ExecStart=/usr/bin/riemann /etc/riemann/riemann.config
+ExecReload=/bin/kill -HUP ${MAINPID}
+ExecStop=/bin/kill -WINCH ${MAINPID}
```
Comment (wickedshimmy) [inline pkg/rpm/riemann.service]: `WINCH`?

**Comment by mfournier** [comment]
IMHO the problem here is not systemd, but the dependency on daemonize.

We can bundle both the sysv initscript and the systemd unit file in the same RPM. And ideally have a conditional postinst script which runs `chkconfig` or `systemctl daemon-reload` depending on the distro version.

But if we want to drop the dependency on daemonize, I think we should change the initscript for ante-systemd distros, and make it fail with a useful error message ("please install daemonize from EPEL6" or whatever).

File: pkg/rpm/riemann.service (hunk 1 above)
Comment (jamtur01) [inline pkg/rpm/riemann.service]: Old habits.

**Comment by jamtur01** [comment]
@mfournier Or rewrite the init script to just not use daemonize?

**Comment by pyr** [comment]
So @mfournier @jamtur01 do we keep this around or close this in anticipation of a follow-up PR dropping the daemonize requirement from init scripts?

**Comment by mfournier** [comment]
daemonize is almost in epel7 ! Please post a couple of upvotes here: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2015-74071f9932 ;)

So this is on tracks for real @pyr, this issue can be closed as far as I'm concerned.

**Comment by pyr** [comment]
@mfournier thanks!

**Pull request closed without merging**